Lib/test/test_pwd.py

```diff
@@ -59,6 +59,8 @@ def test_errors(self):
         self.assertRaises(TypeError, pwd.getpwnam)
         self.assertRaises(TypeError, pwd.getpwnam, 42)
         self.assertRaises(TypeError, pwd.getpwall, 42)
+        # embedded null character
+        self.assertRaises(ValueError, pwd.getpwnam, 'a\x00b')
```

Comment:
I prefer being slightly more explicit:

```suggestion
        self.assertRaisesRegex(ValueError, 'null', pwd.getpwnam, 'a\x00b')
```

Reply:
It would be good if this change was applied to the grp module as well.